src/DependencyInjection/Factory/ContentAdminFactory.php

```diff
@@ -33,6 +36,21 @@ public function getKey()
      */
     public function addConfiguration(NodeBuilder $builder)
     {
+        $builder
+            ->arrayNode('ivory_ckeditor')
+                ->addDefaultsIfNotSet()
+                ->treatFalseLike(['enabled' => false])
+                ->treatTrueLike(['enabled' => true])
+                ->treatNullLike(['enabled' => 'auto'])
+                ->children()
+                    ->enumNode('enabled')
+                        ->values([true, false, 'auto'])
+                        ->defaultValue('auto')
+                    ->end()
+                    ->scalarNode('config_name')->defaultValue('cmf_sonata_admin_integration')->end()
```

Comment:
i think in the original PR we had no default value for this because the ivory bundle provides no default configuration that we want to use. i think it would be clearer if we require this to be explicitly defined if enabled. otherwise people will just have it enabled and then get a js error when the requested configuration does not exist.